core/src/views/header.rs

```diff
@@ -112,4 +112,12 @@ impl<'a> HeaderView<'a> {
         let seal = self.seal();
         seal.into_iter().map(|s| rlp::Rlp::new(&s).data().map(|x| x.to_vec())).collect()
     }
+
+    /// Get view in the seal field of the header.
+    pub fn view(&self) -> Option<u64> {
```

Comment:
How about making the view of the solo consensus be always zero instead of returning `None`?

Reply:
I implemented to return `None` because the view value is initialized to zero for every new height. I wanted to distinguish the solo consensus from the other consensus. And I implemented the compare function like this:
```rust
    /// Compare the number and the view of current block with these of best block
     fn is_new_header_eligible_to_be_best(&self, new_header: &HeaderView<'_>) -> bool {
         let details_of_best_block = self.best_proposal_block_detail();

          (new_header.number(), details_of_best_block.view) > (details_of_best_block.number, new_header.view())
     }
```
If the view of the solo consensus be always zero instead of returening `None`, It seems to add a new condition which is can tell whether it is zero or not, and the return type needs to be changed from `Option<u64>` to `u64`. How about this?

Reply:
There is no difference between returning None and retuning 0.
In your implementation, the view of the solo is always None, so the `is_new_header_eligible_to_be_best` function just compares the block number.

Reply:
I modified it. Thank you : )